plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -239,7 +243,11 @@ private void removeSnapshotFromDockerHub(String repository) throws SnapshotExcep
                 conn.disconnect();
             }
         } catch (IOException e) {
-            LOG.error(e.getLocalizedMessage(), e);
+            LOG.error("Failed to remove {} snapshot from {}. Cause: {}",
+                    repository,
+                    DOCKER_HUB_BASE_URI,
+                    e.getLocalizedMessage(),
+                    e);
```

Comment:
You have 3 "{}", but arguments 4. Maybe delete fourth argument if it is not used?